docs/src/main/sphinx/installation/plugins.md

```diff
@@ -0,0 +1,283 @@
+# Plugins 
+
+Trino uses a plugin architecture to extend its capabilities and integrate with
+various data sources and other systems. Trino includes many plugins as part of
+the binary packages - specifically the `tar.gz` archive and the Docker
+container.
+
+The main plugins implement some the following functionalities:
+
+* [Connectors](/connector)
+* [Access control systems](security-access-control)
+* [Event listeners](admin-event-listeners)
+* Additional [types](/develop/types) and [functions](/develop/functions)
+
+All plugins are optional for your use of Trino because they support a specific
+functionality, that is potentially not needed for your use. Plugins are located
+in the `plugin` folder of your Trino installation and are loaded automatically
+during Trino startup.
+
+(plugins-downloading)=
+## Downloading
+
+Typically, downloading a plugin is not necessary because Trino binaries include
+many plugins as part of the binary packages.
+
+Every Trino release publishes each plugin as a ZIP archive to the [Maven Central
+Repository](https://central.sonatype.com/). Refer to [plugins-list] for details.
+The specific location is derived from the Maven coordinates of each plugin as
+defined in the `pom.xml` of the source code for the plugin.
+
+For example, the PostgreSQL connector plugin can be found in the
+`plugin/trino-postgresql` directory, and the `pom.xml` file contains the
+following identifier section:
+
+```xml
+<parent>
+    <groupId>io.trino</groupId>
+    <artifactId>trino-root</artifactId>
+    <version>470</version>
+    <relativePath>../../pom.xml</relativePath>
+</parent>
+
+<artifactId>trino-postgresql</artifactId>
+<packaging>trino-plugin</packaging>
+```
+
+The Maven coordinates are therefore `io.trino:trino-postgresql:470` with version
+or `io.trino:trino-postgresql` without version. Use this term for a [search to
+locate the
+artifact](https://central.sonatype.com/search?q=io.trino%3Atrino-postgresql).
+
+The coordinates translate into a path to the ZIP archive on the Maven Central
+Repository. Use this URL to download the plugin.
+
+```
+https://repo1.maven.org/maven2/io/trino/trino-postgresql/470/trino-postgresql-470.zip
+```
+
+Availability of plugins from other projects and organizations varies widely, and
+may require building a plugin from source.
+
+(plugins-installing)=
+## Installing
+
+Trino and the installed plugins must use the same version of Trino to be fully
+compatible. For example, a Trino plugin compiled for Trino 470 can not be used
+with older or newer versions of Trino such as Trino 430 or Trino 490.
```

Comment:
That's only true for plugins in the Trino repository, not third party plugins, which are only constrained by the SPI compatibility. There's no mechanism to determine the compatible versions range though

Reply:
I know.. I didnt want to expose that minute detail in the user docs .. but I have to rethink this a bit since I also have to update the dev guide .. I forgot about some of the content we had there..

Reply:
Yeah, people often try building custom plugins by adding a new module in their fork of the Trino repo, instead of a standalone repo, and they copy this check from other plugins, which is not necessary. But this is not a good place to explain this difference.

I'd just remove this section.

Reply:
Cant really remove it since this is an important detail. I reworded it and moved it down in this section to deemphasize.